### PR TITLE
feat(relayer): Add relayer region to the wallet v2 settings

### PR DESCRIPTION
**Pull request by crypblizz8**
Another PR related to #57, this PR is particularly for the `react-wallet-v2` app

[Loom Video](https://www.loom.com/share/aa4c756e71294fa99e4982d983addf01)

Process (Data)
1. Add `relayerRegionURL` + `setRelayerRegionURL` to settings store
2. Call `relayerRegionURL` through `valtio` through `useInitialization()`
3. Pass `relayerRegionURL` through to `createClient()`
4. Track the `prevRelayerURLValue` through `useRef`
5. Re-render when new `relayerURL` value is changed which instantiates a new `createClient()`

Process (Frontend)
1. Add types and utils list to `RelayerRegions` in the data folder
2. Create a `RelayRegionPicker` (Similar to AccountPicker) component.
3. SetState on the `select` value. Data should pass through

### CODE REVIEW

File: wallets/react-wallet-v2/src/data/RelayerRegions.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+type RelayerType = {
+    value: string ;
+    label: string;
+};
+
+/**
+ * Relayer Regions
+ */
+export const REGIONALIZED_RELAYER_ENDPOINTS: RelayerType[] = [
```
Comment (arein) [inline wallets/react-wallet-v2/src/data/RelayerRegions.ts:13]: we also have staging (`staging.relay.walletconnect.com` and `https://staging.us-east-1.relay.walletconnect.com/hello` etc). But that can be in done in a dedicated PR

**Review by arein — APPROVED** [review]

File: wallets/react-wallet-v2/src/data/RelayerRegions.ts (hunk 1 above)
Comment (crypblizz8) [reply wallets/react-wallet-v2/src/data/RelayerRegions.ts:13]: Will add to `dev` in another PR.
